src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Validator/Constraints/ValidAttribute.php

```diff
@@ -31,11 +32,18 @@ class ValidAttribute extends Constraint
 
     public $messageAttributeMotivationNotSet = 'validator.attribute.motivation_not_set';
 
+    public $type = [Constants::TYPE_SAML, Constants::TYPE_OPENID_CONNECT_TNG];
+
     /**
      * @return string
      */
     public function validatedBy()
     {
         return 'valid_attribute';
     }
+
+    public function getDefaultOption()
+    {
+        return 'type';
```

Comment:
I don't get this, does this indicate it should look to the `type` field for the default option value?

Reply:
It is a dev left-over